content/docs/tutorials/data-pipeline.md

```diff
@@ -105,7 +143,7 @@ This SmartModule will do a basic mapping of the JSON input into a SQL statement
 $ fluvio hub download infinyon/json-sql@0.1.0
```

Comment:
Is this repeated now? It looks like we have it above.

Reply:
No. At first occurrence, we add `jolt` to http, then, `json-sql` to sql and `jolt` to sql with a comment that if haven't done that before.